pkgs/by-name/ru/rutabaga_gfx/package.nix

```diff
@@ -42,6 +42,11 @@ stdenv.mkDerivation (finalAttrs: {
     ./darwin-install.patch
   ];
 
+  postPatch = lib.optionalString stdenv.hostPlatform.useLLVM ''
+    substituteInPlace rutabaga_gfx/build.rs \
+      --replace-fail "cargo:rustc-link-lib=dylib=stdc++" "cargo:rustc-link-lib=dylib=c++"
```

Comment:
Can you send a fix for this upstream?  Could check whether the target is LLVM or not.

Reply:
I'm not familiar with `build.rs` and I don't see a way to submit patches to Google.

Reply:
https://chromium.googlesource.com/crosvm/crosvm/+/refs/heads/main/CONTRIBUTING.md

Reply:
I didn't find an issue relating to this so I opened one (https://issuetracker.google.com/issues/392934337) until I have the time to figure out and work on a proper patch.